test/firtool/verilog_equiv.fir

```diff
@@ -0,0 +1,245 @@
+; REQUIRES: yosys
```

Comment:
This seems like more of an integration test to me.

Reply:
Agreed, I think that all of these will be.  I think that having yosis equivalence checking will be a valuable thing, but we should use it very sparingly, and for larger scale integration test sorts of things.

Reply:
Larger test cases will be valuable, but the coverage of firrtl combinatorial ops found ~8 ops with incorrect codegen and 1 bug in scala firrtl.  These small tests are also very fast (1 second range).

Reply:
> Larger test cases will be valuable, but the coverage of firrtl combinatorial ops found ~8 ops with incorrect codegen and 1 bug in scala firrtl. These small tests are also very fast (1 second range).

Sure, but the EmitVerilog tests run in 10s of milliseconds :)

What I'm suggesting is that we split the "find the bugs" phase of this to be an process that is offline from test flows, and keep "verify the bug fixes" online.  The offline process can be really slow, but run against huge corpuses of designs, and be partially manual.  The online tests need to be extremely fast because we'll be running them (and 10's of thousands of their future friends) all the time.

Reply:
Also, the yosis tests will only run for people that have yosis installed, we want the regression tests to be run all the time.

Reply:
The verilator tests in the regression test directory (which check for basic verilator-only functionality) won't run unless one has verilator installed. So there is precedent for that behavior. They, however, run in 80ms each.